CHANGELOG.md

```diff
@@ -15,15 +15,22 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
     _Security_ in case of vulnerabilities.
  -->
 
-## [Unreleased](https://github.com/o1-labs/snarkyjs/compare/1a984089...HEAD)
+## [Unreleased](https://github.com/o1-labs/snarkyjs/compare/97e393ed...HEAD)
 
-> No unreleased changes yet
+> No unreleased changes
+
+## [0.9.8](https://github.com/o1-labs/snarkyjs/compare/1a984089...97e393ed)
+
+### Fixed
+
+- Fix fetching the `access` permission on accounts https://github.com/o1-labs/snarkyjs/pull/851
+- Fixes to `fetchActions` https://github.com/o1-labs/snarkyjs/pull/844 https://github.com/o1-labs/snarkyjs/pull/854 [@Comdex](https://github.com/Comdex)
```

Comment:
the `to` here sounds wrong, not sure if that makes sense here :thinking:

Reply:
I meant multiple different fixes to the same function, but yeah I guess that's not clearly formulated

Reply:
ahh now I understand!